org.scala-refactoring.library/src/main/scala/scala/tools/refactoring/common/tracing.scala

```diff
@@ -21,7 +26,32 @@ trait Tracing {
   def trace(msg: => String)
 }
 
-trait ConsoleTracing extends Tracing {
+object DebugTracing {
+  private val debugStream = {
+    Option(System.getProperty("scala.refactoring.traceFile")).flatMap { fn =>
```

Comment:
When are right about this, no one probably wants to log this refactoring output to the Scala IDE log. I'm fine with the code then. LGTM, I'll let @misto handle the rest.